src/python/CRABClient/Commands/status2.py

```diff
@@ -92,10 +96,15 @@ def __call__(self):
         self.printPublication(publicationEnabled, shortResult['jobsPerStatus'], asourl, asodb,
                               taskname, user, crabDBInfo)
         self.printErrors(statusCacheInfo)
+
         if self.options.summary:
             self.printSummary(statusCacheInfo)
         if self.options.long or self.options.sort:
-            sortdict = self.printLong(statusCacheInfo, quiet = (not self.options.long))
+            # If user correctly passed some jobid CSVs to use in the status --long, self.jobids
+            # will be a list of strings already parsed from the input by the validateOptions()
+            if self.jobids:
+                self.checkUserJobids(statusCacheInfo, self.jobids)
```

Comment:
It's because we haven't called the DB yet so there's nothing to validate against.